citro3d-sys/build.rs

```diff
@@ -66,8 +66,10 @@ fn main() {
         .prepend_enum_name(false)
         .fit_macro_constants(true)
         .raw_line("use ctru_sys::*;")
+        .raw_line("use libc::FILE;")
```

Comment:
About this conflict with `libc::` types... in https://github.com/rust3ds/ctru-rs/pull/186 I thought I had to do something similar, but I don't think anything actually uses the `FILE` type in `ctru-sys`, somehow.

I actually started working on a similar PR with #49 and it looks like I ended up just doing the same `blocklist_type` there ultimately. The reason I haven't undrafted that yet is that it requires https://github.com/rust-lang/rust-bindgen/pull/2836 to avoid conflicting with `ctru-sys`, which hasn't been merged yet either 😢

----

In any case, I think the `raw_line` and `blocklist_type` approach you've got here is fine. From what I can tell, `ctypes_prefix` only applies to fundamental types like `c_int` and `c_char`, not to other more complicated libc types like `FILE`, `timeval` etc. So I think what you have here is about as correct as it can be, and although I would normally expect `.blocklist_file(".*/sys/.*[.]h")` to be sufficient, in this case I guess it's not.